src/fontra/core/clipboard.py

```diff
@@ -64,7 +64,10 @@ def parseGLIF(data: str) -> StaticGlyph | None:
     except GlifLibError:
         return None
     return StaticGlyph(
-        path=pen.getPath(), components=pen.components, xAdvance=ufoGlyph.width
+        path=pen.getPath(),
+        components=pen.components,
+        xAdvance=ufoGlyph.width,
+        anchors=ufoGlyph.anchors,
```

Comment:
This can not work: the anchors need to be converted to Anchor objects. But let me think, it feels like we may need a better/more central way to convert .glif to StaticGlyph.

Reply:
For now, please just insert Anchor conversion code, just like you did in designspace.py. Those two bits of code are different enough that I don't immediately see a good way to refactor it to have more code in common.

Reply:
Thanks. Done.